concrete-core-bench/README.md

```diff
@@ -0,0 +1,13 @@
+# Concrete Core Benchmarks
+
+This library contains benchmarking facilities for the `concrete-core` library.
+
+To execute the benchmark:
+```shell
+RUSTFLAGS="-Ctarget-cpu=native" cargo run --release --features=backend_core -- --bench
```

Comment:
is the --features required here as you have it as the default feature in Cargo.toml?

Or is it considered better practice to give the full set of feature to compile?

Reply:
Well basically, we will have one `backend_*` feature flag for every backends, but we won't be able to all include them in the build. I wanted to show here that you have to activate the feature flag for the backend you want to benchmark. Actually, maybe it would be better to not have `backend_core` in default ? There is nothing special about this guy. What do you think ?

Reply:
I like explicit things and it can show people don't need the backend_core if they don't want it 🙂 

But I'm very new to all the rust conventions so I trust your opinion on that more than mine at the moment

Reply:
Okay then, I removed the default feature flag, and I added a line in the readme to explain better the reason for this feature flag thing ! Thanks !